docs/generate-time-series/go.rst

```diff
@@ -167,18 +166,24 @@ endpoint and returns longitude and latitude as a `geo_point`_ declaration.
     }
 
 Above, the ``getISSPosition()`` function:
-- Uses the `net/http`_ package from the Go standard library to issue an 
-HTTP GET request to the API endpoint 
-- Implements some basic error handling and checks to see whether the
-response code is in the 200 range
-- Reads the response body and unmarshals the JSON into the defined 
-struct ``issInfo`` 
-- Formats the return string and returns it 
-
-Then in the main function, call the ``getISSPosition()`` function and print 
+
+.. rst-class:: open
```

Comment:
@infoverload when a list looks better in the source file separated by empty newlines, I add this RST directive which formats the list in HTML with paragraph margins on the list items (to mirror the formatting)